### DISCUSSION TITLE
Current version 1.0 or 0.3.4

### DISCUSSION THREAD

**Discussion by dirkreiter** [zhonge/cryodrgn#116] [General]
Hi,
I am wondering whether the current version is 1.0 or 0.3.4? On the main page the new features of version 1.0 are mentioned and 0.3.4 is listed as a previous version. However, in the version file 0.3.4 is listed as the current release and this also seems to be the newest version available for installation. Just wondering if it still makes sense to install 0.3.4 or whether 1.0 is either already available or will be available very soon?

Thank you

**Comment by zhonge** [comment]
Hi! Thanks for the question. I'm in the process of writing the documentation describing the new features in version 1.0 before the official release. Apologies for the confusion (and I should have really done this sooner to avoid the ambiguity). I will go ahead and release the current version as 1.0-beta, and I'll update here when I get a chance to fully release version 1.0.

**Reply by zhonge** [reply]
I just released version 1.0.0-beta. Let me know if you run into anything. I will update again when the official version 1.0 is released.

**Reply by dirkreiter** [reply]
Thank you so much for the super-fast response and the release of the Beta! I have just installed it and will let you know if we experience any issues.
